bus-mapping/src/evm/opcodes/logs.rs

```diff
@@ -100,12 +99,12 @@ fn gen_log_step(
 
     // generates topic operation dynamically
     let topic_count = match exec_step.exec_state {
-        ExecState::Op(op_id) => (op_id.as_u8() - OpcodeId::LOG0.as_u8()) as usize,
+        ExecState::Op(op_id) => op_id.postfix().unwrap(),
```

Comment:
How about:
```suggestion
        ExecState::Op(op_id) => op_id.postfix().expect("your expectation TODO") as usize,
```

So that we avoid the two following `as usize` changes?